pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -447,13 +460,36 @@ func encodeEventLog(attestCtx *attestContext, quoteMsg *attest.ZAttestQuote) err
 		if event.Index == 8 || event.Index == 9 {
 			tpmEventLog.EventDataString = string(event.Data)
 		} else {
-			tpmEventLog.EventDataString = "Not Applicable"
+			tpmEventLog.EventDataString = notApplicableString
 		}
 		quoteMsg.EventLog = append(quoteMsg.EventLog, tpmEventLog)
 	}
 	return nil
 }
 
+//deduplicateEventLog removes duplicates String/Binary
+func deduplicateEventLog(quoteMsg *attest.ZAttestQuote) {
+	for _, el := range quoteMsg.EventLog {
+		if el.EventDataString != notApplicableString {
```

Comment:
Instead of this, why not just skip setting it in encodeEventLog()?

Reply:
I put comment instead of encoding of EventDataString for now